src/macaron/slsa_analyzer/ci_service/github_actions/github_actions_ci.py

```diff
@@ -1,69 +1,38 @@
-# Copyright (c) 2022 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2022 - 2024, Oracle and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
 
 """This module analyzes GitHub Actions CI."""
 
+from __future__ import annotations
```

Comment:
I wonder which annotation feature are we using that requires this import.

Reply:
I used this import at some point not to require defining classes in a specific order. But that's not needed anymore. See commit [0ac920f](https://github.com/oracle/macaron/pull/673/commits/0ac920f981e277943b32b212350e1b1c65233f41)